.rubocop_local.yml

```diff
@@ -1,3 +1,6 @@
+AllCops:
+  TargetRubyVersion: 2.5
```

Comment:
This should not be necessary as the default coming from the shared manageiq-style, is this value - https://github.com/ManageIQ/manageiq-style/blob/70ddb2400036f31404c58a143aff8ed82645b1a0/styles/base.yml#L10

Reply:
Strange, I saw differences in the linting behaviour when I include the all cops. I'll remove it and retest.